src/components/shared/__tests__/ChatListItem.test.tsx

```diff
@@ -69,8 +70,18 @@ describe('[ChatListItem] interaction', () => {
   });
 
   it('should fireEvent when peer image is pressed', () => {
-    const touchPeerImage = testingLib.getByTestId('peer_image');
+    // If I don't add testID to props (line 50), line 75~77 doesn't work.
+    // 1) Should I add testID to props (line 50)? Then, Why is there testID in ChatListIten.tsx?
```

Comment:
To help you understand this, each component test works independently. Therefore, you don't have to imagine whether the `props` are passed down to `ChatListItem` from `Chat` correctly. You can just concentrate on the `unit` test that's in `ChatListItem`.

Reply:
You should add `testID` to props to if you want to grab the `element` with the `testID` or you can just grab it with `type` ([reference](https://testing-library.com/docs/dom-testing-library/api-queries)).

Reply:
Then,
`<TouchableOpacity testID={testID} onPress={onPressPeerImage}>` line 135  in`ChatListItem.tsx`

This testID is not needed for test code in `ChatListItem.test.tsx` now?

Reply:
What I meant was there is other way to grab the `element` rather than the `testID`. Since you are using `testID` to grab the `element` to test, it is necessary :)